### PR TITLE
Safari 15.5

### PR DESCRIPTION
**Pull request by willyaranda**

### CODE REVIEW

File: resources/user-agents/browsers/safari/safari-15-0-to-15-4.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "division": "Safari #MAJORVER#.#MINORVER#",
-    "versions": ["15.4", "15.3", "15.2", "15.1", 15],
+    "versions": ["15.5", "15.4", "15.3", "15.2", "15.1", 15],
```
Comment (asgrim) [inline resources/user-agents/browsers/safari/safari-15-0-to-15-4.json:3]: Do we have any example UA strings for these?

Comment (willyaranda) [reply resources/user-agents/browsers/safari/safari-15-0-to-15-4.json:3]: just pushed

**Comment by asgrim** [comment]
@willyaranda hmm - can you rebase rather than merge-up for branch updates please? If you're not confident with rebase, let me know and I'll rebase + merge. Thank you

**Comment by willyaranda** [comment]
@asgrim done!

**Comment by asgrim** [comment]
Great, thank you so much!

**Review by asgrim — APPROVED** [review]

**Pull request merged**